tests/Microplate/CoordinatesTest.php

```diff
@@ -130,40 +166,38 @@ public function testPositionWells(CoordinateSystem $coordinateSystem, array $wel
      *   }
      * > $paddedWells
      */
-    public static function dataProviderPaddedWells(): array
+    public static function dataProviderPaddedWells(): iterable
     {
-        return [
-            '12Wells' => [
-                new CoordinateSystem4x3(),
-                [
-                    ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
-                    ['paddedCoordinates' => 'C4', 'row' => 'C', 'column' => 4],
-                ],
+        yield '12Wells' => [
+            new CoordinateSystem4x3(),
+            [
+                ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
+                ['paddedCoordinates' => 'C4', 'row' => 'C', 'column' => 4],
             ],
-            '48Wells' => [
-                new CoordinateSystem8x6(),
-                [
-                    ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
-                    ['paddedCoordinates' => 'F8', 'row' => 'F', 'column' => 8],
-                ],
+        ];
+        yield '48Wells' => [
+            new CoordinateSystem8x6(),
+            [
+                ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
+                ['paddedCoordinates' => 'F8', 'row' => 'F', 'column' => 8],
             ],
-            '96Wells' => [
-                new CoordinateSystem12x8(),
-                [
-                    ['paddedCoordinates' => 'A01', 'row' => 'A', 'column' => 1],
-                    ['paddedCoordinates' => 'C05', 'row' => 'C', 'column' => 5],
-                    ['paddedCoordinates' => 'H12', 'row' => 'H', 'column' => 12],
-                    ['paddedCoordinates' => 'D10', 'row' => 'D', 'column' => 10],
-                ],
+        ];
+        yield '96Wells' => [
+            new CoordinateSystem12x8(),
+            [
+                ['paddedCoordinates' => 'A01', 'row' => 'A', 'column' => 1],
+                ['paddedCoordinates' => 'C05', 'row' => 'C', 'column' => 5],
+                ['paddedCoordinates' => 'H12', 'row' => 'H', 'column' => 12],
+                ['paddedCoordinates' => 'D10', 'row' => 'D', 'column' => 10],
             ],
-            '2x16Wells' => [
-                new CoordinateSystem2x16(),
-                [
-                    ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
-                    ['paddedCoordinates' => 'B2', 'row' => 'B', 'column' => 2],
-                    ['paddedCoordinates' => 'M1', 'row' => 'M', 'column' => 1],
-                    ['paddedCoordinates' => 'K2', 'row' => 'K', 'column' => 2],
-                ],
+        ];
+        yield '2x16Wells' => [
+            new CoordinateSystem2x16(),
+            [
+                ['paddedCoordinates' => 'A1', 'row' => 'A', 'column' => 1],
+                ['paddedCoordinates' => 'B2', 'row' => 'B', 'column' => 2],
+                ['paddedCoordinates' => 'M1', 'row' => 'M', 'column' => 1],
+                ['paddedCoordinates' => 'K2', 'row' => 'K', 'column' => 2],
             ],
```

Comment:
Hier wuerde es noch sinn machen die 6x8 Daten noch in dataProviderWells mitzugeben